contracts/protocol/pool/PoolConfigurator.sol

```diff
@@ -186,6 +188,29 @@ contract PoolConfigurator is VersionedInitializable, IPoolConfigurator {
         );
     }
 
+    /// @inheritdoc IPoolConfigurator
+    function configureReserveAsAuctionCollateral(
+        address asset,
+        bool auctionEnabled,
+        uint256 auctionRecoveryHealthFactor
```

Comment:
why are we setting this here? can't we set it in the `setAuctionConfiguration` ?